shared/mk/tendra.makedefs.mk

```diff
@@ -41,17 +41,14 @@ VERSION?=	${UNAME_REVISION}
 
 .if ${SYSTEM} == Linux
 
-.if exists(/lib/libc.so.6)
-LDD_BLURB?= /lib/libc.so.6 2>&1
-.else
-LDD_BLURB?= ldd --version  2>&1
-.endif
+LDD_BLURB?= (ldd --version  2>&1 || /lib/libc.so.6 2>&1)
```

Comment:
Okay, so we'll find out if we encounter any Linux systems in the future where we don't have `ldd --version` around. I know it seems odd, but I did encounter that! I think it was for eglibc.